prime-dotnet-webapi/ViewModels/Enrollee/EnrolleeAbsenceViewModel.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Prime.ViewModels
+{
+    public class EnrolleeAbsenceViewModel
+    {
+        public int Id { get; set; }
+
+        public int EnrolleeId { get; set; }
```

Comment:
Do we need Id or EnrolleeId on the VM? Is the frontend actually using them?

Reply:
We need Id, but can infer enrollee Id.